### DISCUSSION TITLE
Incorrect version ordering (used renovate:latest)

### DISCUSSION THREAD

**Discussion by pshakhov** [renovatebot/renovate#11742] [Request Help] [answered]
While updating dependencies renovate up Apache `commons-io` from `2.9.0` to `20030203.000550` (latest update in 2005) instead `2.11.0`. Its very dangerous change. Or its my renovate config can be incorrect?

https://mvnrepository.com/artifact/commons-io/commons-io

**Comment by viceice** [comment, marked as answer]
that's a bad version published, renovate should ignore those if you use our `config:base` preset.

https://github.com/renovatebot/renovate/blob/5731da899be9ac174886c48a5fb124a99ac6f688/lib/config/presets/internal/workarounds.ts#L16-L24

So what's your config?
